tools/fuchsia/flutter/flutter_build_config.gni

```diff
@@ -26,7 +26,7 @@ flutter_debug_build_cfg = {
 # # profile aot runner is built without asserts.
 # flutter_aot_debug_build_cfg = {
 #   runtime_meta = "//flutter/shell/platform/fuchsia/flutter/meta/aot_runtime"
```

Comment:
Do we need all these extra commented-out configs or should we delete them until we find a need for them in Engine? If we're only planning on using the SDK for making Fuchsia tests run, will there ever be a need for these configs?

Reply:
Yes, we will need them, but I didn't get around to validating them yet. I'm not yet as familiar with all of these modes, but this should be updated sooner rather than later. I don't THINK it is critical to publish this PR (but we'll see).